bql/planner/planner_test.go

```diff
@@ -670,6 +670,16 @@ func TestPlannerQuery(t *testing.T) {
 			nBindings: 3,
 			nRows:     4,
 		},
+		{
+			q: `SELECT ?o, ?o_type
+				FROM ?test
+				WHERE {
+					?s ?p ?o TYPE ?o_type
+				}
+				HAVING (?s = /u<joe>) OR (?s = /l<barcelona>) OR (?s = /u<alice>);`,
+			nBindings: 2,
+			nRows:     2,
+		},
```

Comment:
This change is in fact added with the PR #119 ("Fix bug of TYPE bindings with non-node objects and AT bindings with immutable triples"), appearing here because of the rebase.
The intent of this test there is to ensure that we are not having a deadlock when trying to extract the `TYPE` from objects that are literals or predicates, as it was the case before. At this moment the expected behavior is to skip these triples, which will be changed with PR #126 later.
For that, the simplest way I found to test the `TYPE` extraction against all the three types of objects (nodes, predicates and literals) without querying the entire graph and returning 23 rows was restricting the `?s` to the three above. This way it returns only 2 rows and it is easier for anyone looking at the test to look at the graph and verify that the query returns the expected number of rows.